cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2733,7 +2733,38 @@ action.cyences_notable_event_action.attacker_drilldown = `cs_o365` sourcetype="o
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
 
-
+[O365 - OneDrive or SharePoint File Sharing with External User]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 4 * * *
+description = This alert will pinpoint when file sharing with external user usein Onedrive and Sharepoint. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -4h@h
+dispatch.latest_time = +4h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@mWorkload IN (OneDrive, SharePoint) Operation IN ("SharingSet", "AccessRequestApproved", "SharingRestored", "AnonymousLinkCreated", "SharingInvitationCreated", "SharingInvitationUpdated", "AddedToSecureLink", "SharingPolicyChanged") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") \
+| stats values(Operation) AS Operations, values(EventData) AS EventData, values(ItemType) as ItemType,values(EventSource) as EventSource latest(_time) as LastTime earliest(_time) as FirstTime by SourceFileName, UserId \
+| `cs_human_readable_time_format(FirstTime)` \
+| `cs_human_readable_time_format(LastTime)` \
+| eval cyences_severity = "high" \
+| table UserId, SourceFileName,EventSource,ItemType, Operations, FirstTime, LastTime, EventData, cyences_severity \
```

Comment:
Keep the cyences_severity field before the Operations field as it is more important.

And its a good idea to be consistent in terms of spaces and comma:
UserUd, SourceFileName, EventSource, ItemType, ...